src/remarx/app/utils.py

```diff
@@ -5,21 +5,78 @@
 import contextlib
 import pathlib
 import tempfile
-from collections.abc import Generator
+import webbrowser
+from collections.abc import AsyncGenerator, Generator
+from contextlib import asynccontextmanager
 
-from marimo._cli import cli
+import marimo as mo
+import uvicorn
+from fastapi import FastAPI
+from fastapi.responses import RedirectResponse
 
 # Does this class have a public facing type definition?
 from marimo._plugins.ui._impl.input import FileUploadResults
 
-from remarx.app import ui
+import remarx
+
+# Server configuration
+HOST = "localhost"
+PORT = 8000
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI) -> AsyncGenerator[None, None]:
+    """Lifespan context manager to open browser when server starts"""
+    webbrowser.open(f"http://{HOST}:{PORT}/")
+    yield
+
+
+async def redirect_root() -> RedirectResponse:
+    """Redirect root path to corpus-builder"""
+    return RedirectResponse(url="/corpus-builder", status_code=302)
```

Comment:
I thought there'd be a more efficient way to declare this in the app setup, but seems fine. We might want to tweak the comment for future us
```suggestion
async def redirect_root() -> RedirectResponse:
    """Redirect root path to corpus-builder, since app currently has no home page"""
    return RedirectResponse(url="/corpus-builder", status_code=302)
```